### ISSUE TITLE
false positive (flying) when using a water elevator

### ISSUE THREAD

**Issue by tomaste** [gravitylow/AntiCheat#269]
When users use a water elevator that is surrounded by glass, they get kicked for flying.

**Comment by gravitylow** [comment]
Is this in a boat?

**Comment by tomaste** [comment]
no, this can be replicated on my server in any stream of water.  Initially I thought it was just a water elevator, but it appears that when swimming _up_ any stream of water greater than 3 blocks, it kicks.
